service/src/config/env.go

```diff
@@ -27,7 +27,7 @@ func NewEnv() *Env {
 	return &Env{
 		Version: Version{
 			Tag:    tag,
-			Commit: commit,
+			Commit: commit[0:7],
```

Comment:
1. We should handle case when `commit` can be less than 7 characters for example for dev env it throws error right now
2. Whe have you decided to use `0:7` as I see it returns 7 characters instead of 8, I would say it is better to return 8 as in Frontend

Reply:
btw I think right now that it is better to introduce some additional field in the `Version` struct something that would be called `Label` and add full name for frontend and for backend to this place, just to simplify change in both places in the future

Reply:
1. Addressed it.
2. To match short hash displayed in Github which is 7 chars.

Reply:
2. yeah it makes sense